WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/InPersonPaymentsLearnMore.swift

```diff
@@ -9,14 +9,8 @@ struct InPersonPaymentsLearnMore: View {
                 .resizable()
                 .foregroundColor(Color(.textSubtle))
                 .frame(width: iconSize, height: iconSize)
-            AttributedText(Localization.learnMore)
+            Text(Localization.learnMore)
```

Comment:
I'm curious, is removing the link to Learn More deliberate?

Reply:
See https://github.com/woocommerce/woocommerce-ios/pull/5030#discussion_r713482130

Reply:
If I understand correctly, this is fixed in #5068 . Should it still be part of the diff?

Reply:
No. Now that this is merged, I'll rebase and resolve the expected conflict

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/5030/commits/5cd52b50f282a174a40d7b4962dacc6f58c1143b